### DISCUSSION TITLE
installing cuspatial under pipenv

### DISCUSSION THREAD

**Discussion by matthewjmuscat** [pypa/pipenv#5821] [Q&A]
I am currently attempting to install cuspatial under pipenv. I have brought the question up under the cuspatial github (https://github.com/rapidsai/cuspatial/discussions/1243), but there doesn't seem to be a solution handy from the dev crew, so I thought I would post this question under the pipenv github. 

TLDR other post. I attempted to install cuspatial using the following commands:

```
pipenv install cuspatial-cu12 --index https://pypi.nvidia.com 
pipenv install --index https://pypi.nvidia.com "cuspatial-cu12" 
pipenv install --index https://pypi.nvidia.com cuspatial-cu12 
```

to no avail. See the other post for the full error returned. I believe this may be an issue similar to that of https://github.com/pypa/pipenv/issues/4961, although I am not really certain. As far as I understand, pipenv uses only PEP 503 standardized repos, so I am unsure if this is the cause of the problem or not.

Eventually I was able to install pytorch successfully under pipenv using the following command:

`pipenv install --index https://download.pytorch.org/whl/cu118/ "torch==2.0.1+cu118"
`

and since cuspatial seemed to be a 3rd party package available under the NVIDIA Python Package Index, I thought I had to install the cuspatial package in a similar fashion by manually specifying the source using the --index option when using the install command.

Does anyone know how to install cuspatial under a pipenv environment? If so what is the correct command? 

Thanks.

**Comment by matteius** [comment]
https://github.com/rapidsai/cuspatial/discussions/1243#discussioncomment-6622239
